tests/testthat/setup.R

```diff
@@ -4,5 +4,6 @@ require(limma)
 require(DGEobj)
 require(DGEobj.plots)
 require(DGEobj.utils)
+require(dplyr)
```

Comment:
can we call dplyr through double colon operator :: for the need call only instead of adding it to the setup, as long as it is just one call?